swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -173,7 +221,9 @@ class Unit extends TUnit {
  */
 predicate isUnreachableInCall(Node n, DataFlowCall call) { none() }
 
-newtype LambdaCallKind = TODO_TLambdaCallKind()
+newtype LambdaCallKind =
+  TYieldCallKind() or
+  TLambdaCallKind()
```

Comment:
I forgot to remove these when I ripped out the guts of the Ruby files. Do you mind replacing this with `TODO_TLambdaCallKind()` or similar again?

Reply:
I'll do that as a separate PR

Reply:
Actually just saw there's an autoformat failure so I'll fix it in this